src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -100,38 +147,38 @@ public function normalize($object, $format = null, array $context = [])
         $attributesMetadata = $this->classMetadataFactory ? $this->classMetadataFactory->getMetadataFor($class)->getAttributesMetadata() : null;
         $maxDepthHandler = $context[self::MAX_DEPTH_HANDLER] ?? $this->defaultContext[self::MAX_DEPTH_HANDLER] ?? $this->maxDepthHandler;
 
-        foreach ($attributes as $attribute) {
+        foreach ($attributes as $attributeName) {
```

Comment:
I'd suggest not changing this to reduce merge conflicts with lower branches

Reply:
yeah i see. i am a fan of good naming, but merging lower branches is more important.